### PR TITLE
[TTAHUB-2830] Require that CSV is in UTF-8 encoding

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

The default encoding of CSV files in Excel is [Windows-1252](https://en.wikipedia.org/wiki/Windows-1252). This is an outdated encoding and as far as I can tell is being replaced by UTF-8. This change ensures that the user is creating their CSV as CSV UTF-8.

**NOTE:** This historical data cleanup for this will be a different ticket.

image

image

## How to test

Import the CSV with the default windows csv encoding you should see an error. Then import a CSV with the CSV UTF-8 **attached to the JIRA**. It should import 113 records and all Spanish chars should show up correctly.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2830

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/Admin/components/CsvImport.js
Comment (thewatermethod) [inline frontend/src/pages/Admin/components/CsvImport.js:264]: I'd rather we didn't disable this, it's annoying since you have to refresh to resubmit, even if you change the file, plus the color contrast is poor.

Comment (AdamAdHocTeam) [reply frontend/src/pages/Admin/components/CsvImport.js:264]: My concern here is they can still upload the bad file. Is there another way we can prevent this?

When you say "refresh to submit", all I had to do is change the selected file to enable the button.

Comment (AdamAdHocTeam) [reply frontend/src/pages/Admin/components/CsvImport.js:264]: I guess we can just do this check once when they click the upload button instead.

**Review by thewatermethod — APPROVED** [review]
Looks good. We should probably wait for CI to finish before merging.

File: frontend/src/pages/Admin/components/CsvImport.js
Comment (nvms) [inline frontend/src/pages/Admin/components/CsvImport.js:100]: Do we care to handle possible rejection here, or do we think it's too unlikely to bother with?

**Review by nvms — APPROVED** [review]
Looks good

File: frontend/src/pages/Admin/components/CsvImport.js
Comment (AdamAdHocTeam) [reply frontend/src/pages/Admin/components/CsvImport.js:100]: What do you mean by that?

**Review by nvms — APPROVED** [review]

**Review by kryswisnaskas — APPROVED** [review]
Looks good! 👍

**Pull request merged**
